tests/integration/api/controllers/diagram/test_otm_controller_diagram.py

```diff
@@ -48,3 +63,105 @@ def test_create_project_validation_error(self, project_id: str, project_name: st
         assert len(res_body['errors']) == errors_expected
         for e in res_body['errors']:
             assert len(e['errorMessage']) > 0
+
+    @responses.activate
+    def test_create_project_no_diag_file(self):
+        # Given a project_id and name
+        project_id: str = 'project_A_id'
+        project_name: str = 'project_A_name'
+
+        # And the request files
+        mapping_file = get_byte_data(test_resource_paths.default_drawio_mapping)
+
+        # When I do post on drawio endpoint
+        files = {'diag_file': None, 'default_mapping_file': mapping_file}
+        body = {'diag_type': 'DRAWIO', 'id': project_id, 'name': project_name}
+        response = client.post(get_url(), files=files, data=body)
+
+        # Then the OTM is returned inside the response as JSON
+        assert_bad_request_response(response)
+        assert_bad_request_body_response(response, 'HTTPException',
```

Comment:
Here we are not checking the error message: "Error in field 'diag_file' located in 'body'. Field required"